Source/GUI/MemWatcher/MemWatchWidget.cpp

```diff
@@ -273,6 +273,43 @@ void MemWatchWidget::onMemWatchContextMenuRequested(const QPoint& pos)
   connect(copy, &QAction::triggered, this, [this] { copySelectedWatchesToClipBoard(); });
   contextMenu->addAction(copy);
 
+  if (index != QModelIndex())
+  {
+    MemWatchEntry* entry = m_watchModel->getEntryFromIndex(index);
+    if (entry->isBoundToPointer())
+    {
+      QMenu* copyAddrSubmenu = contextMenu->addMenu(tr("Copy add&ress..."));
+      QAction* copyPointer = new QAction(tr("Copy &base address..."), this);
+      connect(copyPointer, &QAction::triggered, this,
+              [this, entry] { copyAddressToClipboard(entry->getConsoleAddress()); });
+      copyAddrSubmenu->addAction(copyPointer);
+      for (int i = 0; i < static_cast<int>(entry->getPointerLevel()); ++i)
+      {
+        std::string strAddressOfPath = entry->getAddressStringForPointerLevel(i + 1);
+        if (strAddressOfPath == "???")
+          break;
```

Comment:
Instead of testing strings, which is not very robust, could change in the future, and technically slower, why not verify the address directly?
```cpp
if (!entry->getAddressForPointerLevel(level))
    break;
```